spyder/plugins/completion/kite/providers/document.py

```diff
@@ -155,7 +155,7 @@ def convert_completion_request(self, response):
     def request_hover(self, params):
         text = self.opened_files.get(params['file'], "")
         md5 = hashlib.md5(text.encode('utf-8')).hexdigest()
-        path = str(params['file'])
+        path = params['file']
```

Comment:
Should we use `to_text_string` or something? Why was the `str` there on the first place?

Reply:
I'm not sure why it had an `str` before, but this also was causing an encoding error (probably also caused because of the change I did over the `path_as_uri` method where now it returns an `unicode` representation of the uri):

```traceback
  File "C:\Users\Daniel\Documents\Spyder\spyder\spyder\plugins\completion\kite\client.py", line 131, in perform_request
    response = handler(params)
  File "C:\Users\Daniel\Documents\Spyder\spyder\spyder\plugins\completion\kite\decorators.py", line 20, in wrapper
    params = req(self, *args, **kwargs)
  File "C:\Users\Daniel\Documents\Spyder\spyder\spyder\plugins\completion\kite\providers\document.py", line 158, in request_hover
    path = str(params['file'])
UnicodeEncodeError: 'ascii' codec can't encode character u'\xed' in position 21: ordinal not in range(128)

```

Reply:
Not sure either, then we are good to go :-)